core/src/edu/cornell/gdiac/main/PlayerModel.java

```diff
@@ -196,6 +207,16 @@ public boolean isFrozen() {
    */
   public void setFrozen(boolean value) {
     // TODO P3 update frozen and apply physics, also visually change player in some way
+    isFrozen = value;
+    if (isFrozen){
+      setDensity(getDensity() * 3);
+      setFixedRotation(true);
+      color = Color.BLUE;
```

Comment:
Tinting blue is a nice choice to indicate the character being frozen, great job!